### PR TITLE
VZ-9332: Add network policies for Capi

### PR DESCRIPTION
**Pull request by mgianatagh**

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml [hunk 1]
```diff
@@ -796,3 +796,85 @@ spec:
...
+      control-plane: controller-manager
+  policyTypes:
+    - Ingress
+  ingress:
+    - ports:
+      - port: 9443
+        protocol: TCP
+---
+# Network policy for capi-ocne-bootstrap-controller-manager
+# Ingress:
```
Comment (bobdonat) [inline platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml:822]: should there be a description here for the ingress

File: platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml [hunk 2]
```diff
@@ -796,3 +796,85 @@ spec:
...
+      control-plane: controller-manager
+  policyTypes:
+    - Ingress
+  ingress:
+    - ports:
+        - port: 9443
+          protocol: TCP
+---
+# Network policy for capi-ocne-control-plane-controller-manager
+# Ingress:
```
Comment (bobdonat) [inline platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml:842]: same comment

File: platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml [hunk 3]
```diff
@@ -796,3 +796,85 @@ spec:
...
+      control-plane: controller-manager
+  policyTypes:
+    - Ingress
+  ingress:
+    - ports:
+        - port: 9443
+          protocol: TCP
+---
+# Network policy for capoci-controller-manager
+# Ingress:
```
Comment (bobdonat) [inline platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml:862]: same comment

File: platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml [hunk 4]
```diff
@@ -364,7 +364,7 @@ spec:
         - port: 9443
           protocol: TCP
 {{- end }}
-{{- if .Values.argoCd.enabled}}
+{{- if .Values.argocd.enabled}}
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply platform-operator/helm_config/charts/verrazzano-network-policies/templates/thirdparty-networkpolicy.yaml:367]: Yes, it is `argued` for the network policy values.yaml.  And I noticed that my IDE reported `argdCd` is unknown.

**Review by bobdonat — APPROVED** [review]

**Pull request merged**